src/app/app.routes.js

```diff
@@ -57,7 +57,9 @@ angular.module('librarium').config([
         url: '/signup',
         views: {
           main: {
-            templateUrl: '../app/components/users/views/registration.html'
+            templateUrl: '../app/components/users/views/registration.html',
+            controller: 'RegistrationController',
+            controllerAs: 'regCtrl'
```

Comment:
For future reference you can also use ` controller: 'RegistrationController as regCtrl'` you can leave this as is, just so you know in our project we use this convention